### PR TITLE
feat: hermes-relayer config

### PR DESCRIPTION
**Pull request by jlehtimaki**
This PR enables Hermes to be run on Kubernetes on demand and with ease of setup.

### CODE REVIEW

File: charts/hermes-relayer/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Hermes Relayer
+
+## Prerequisites
+
+Kubernetes 1.16+
+
+## Get Repository Info
+
+```console
+helm repo add prometheus-community https://archway-network.github.io/helm-charts
```
Comment (kayano) [inline charts/hermes-relayer/README.md]: Why do we have `prometheus-community` here?

File: charts/hermes-relayer/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          volumeMounts:
+            - name: keys
+              mountPath: "/home/hermes/.hermes/keys"
+            - name: configfile
+              mountPath: "/home/hermes/.hermes/"
+            - name: chain2mnenomic
+              readOnly: true
+              mountPath: /etc/mnemonic
+      containers:
+        - name: {{ .Chart.Name }}
```
Comment (kayano) [inline charts/hermes-relayer/templates/deployment.yaml]: Why not just `relayer` for the name? Easier to use it later I think

File: charts/hermes-relayer/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          image: "{{.Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion}}"
+          imagePullPolicy: {{.Values.image.pullPolicy}}
+          {{- with .Values.containerSecurityContext }}
```
Comment (kayano) [inline charts/hermes-relayer/templates/deployment.yaml]: it should be move before https://github.com/archway-network/helm-charts/pull/3/files#diff-eea9971305cd2a5fcd75b1a4355582ae64d8e634615aee0179a0be79a2f150bdR87

File: charts/hermes-relayer/values.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+replicaCount: 1
+name: hermes-relayer
+image:
+  repository: ghcr.io/informalsystems/hermes
+  pullPolicy: Always
+  tag: ""
+imagePullSecrets: []
```
Comment (kayano) [inline charts/hermes-relayer/values.yaml:7]: Where is it used?

File: charts/hermes-relayer/values.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+replicaCount: 1
+name: hermes-relayer
```
Comment (kayano) [inline charts/hermes-relayer/values.yaml]: Is it used?

File: charts/hermes-relayer/values.yaml (hunk 4 above)
Comment (jlehtimaki) [reply charts/hermes-relayer/values.yaml:7]: It's not used but it can be if needed

**Review by kayano — APPROVED** [review]

**Pull request merged**